dev/devicelab/lib/microbenchmarks.dart

```diff
@@ -9,11 +9,16 @@ import 'dart:io';
 /// Reads through the print commands from [process] waiting for the magic phase
 /// that contains microbenchmarks results as defined in
 /// `dev/benchmarks/microbenchmarks/lib/common.dart`.
+///
+/// If you are using this outside of microbenchmarks, ensure you print a single
+/// line with `╡ ••• Done ••• ╞` to signal the end of collection.
```

Comment:
Optional for this PR, but should we have a TODO to pull this out into like https://github.com/flutter/flutter/blob/master/dev/devicelab/lib/framework/utils.dart (or somewhere better than a utils library...) ?

Reply:
I typically like smaller CLs. We can file a cleanup / tech-debt to do just that.

Reply:
filed: https://github.com/flutter/flutter/issues/154449